kuma/static/js/components/modal.js

```diff
@@ -0,0 +1,88 @@
+(function() {
+    'use strict';
+
+    if (!window.mdn) {
+        window.mdn = {};
```

Comment:
This code would not be needed if the modal.js wasn't `async` loaded.

Reply:
True, but I was also following a pattern that is used in other places in the codebase. Perhaps it is moot at this stage and really simply is not needed. Once we mark it as `defer` would you vote to simply remove these lines?